### ISSUE TITLE
Where is the small LoRA model at not this 2.43GB thing?

### ISSUE THREAD

**Issue by DarkAlchy** [Linaqruf/kohya-trainer#42]
I use automatic1111, and he expects a .pt and the 60-150mb file not an entire model of 2.43GB.  I f I wanted that I would just use Dreambooth.

**Comment by Linaqruf** [comment]
you're using wrong notebook lol

**Comment by DarkAlchy** [comment]
Oops :(  What is the right one?

**Comment by Linaqruf** [comment]
[Fine-tuning method](https://github.com/Linaqruf/kohya-trainer/blob/main/kohya-LoRA-finetuner.ipynb), or [Dreambooth method](https://github.com/Linaqruf/kohya-trainer/blob/main/kohya-dreambooth.ipynb)

**Comment by DarkAlchy** [comment]
What is the difference between those on Colab?

Thank you.

**Comment by Linaqruf** [comment]
Dreambooth method
- using dreambooth data loader
- prior loss weight 1.0
- abillity to use reg images to prevent overfitting 
- train with concept (or caption as well) by renaming your folder with <repeats>_<concept> <class>
- you need to add initial token/dreambooth-like trigger prompt to your prompt

Finetune method
- traditional native training
- train with captions
- can do buckets and latents outside training and save the latents as .npz, 
- you only need to load dataset, load caption inside json, and start training
- no need to use trigger prompt like dreambooth

**Comment by DarkAlchy** [comment]
I like it.  I really liked DB in 1.5 but the methods internally DB would be at the top of the food chain and LoRA/Hypernetwork next and TI at the bottom.  If the model has no clue about something and I want to make it know about it then DB is the way to go from everything I have seen, and read.
